coa-helm/templates/tes.yaml

```diff
@@ -43,6 +43,10 @@ spec:
               value: {{ .Values.config.dockerInDockerImageName }}
             - name: UsePreemptibleVmsOnly
               value: {{ .Values.config.usePreemptibleVmsOnly | quote}}
+            - name: MarthaUrl
+              value: {{ .Values.config.drsUrl }}
+            - name: CromwellDrsLocalizerImageName
+              value: {{ .Values.cromwell.image | replace "cromwell" "cromwell-drs-localizer" }}
```

Comment:
Note that DRS resolution in TES isn't tested yet because TES is not working in the app.